teamsyncd/teamsyncd.cpp

```diff
@@ -9,6 +10,14 @@
 using namespace std;
 using namespace swss;
 
+bool received_sigterm = false;
+
+void sig_handler(int signo)
```

Comment:
>signo [](start = 21, length = 5)

the same

Reply:
The Signal handler is only registered only for SIGTERM now. The handler will be called only on SIGTERM signal.